Sasquatch/SasquatchObjC/AppDelegate.m

```diff
@@ -68,7 +68,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
 // Customize App Center SDK.
 #pragma clang diagnostic ignored "-Wpartial-availability"
-  if ([[NSProcessInfo processInfo] operatingSystemVersion].majorVersion >= 10) {
+  if (@available(iOS 10.0, *)) {
```

Comment:
Not necessary.

Reply:
Do you mean remove if condition or roll back the change?

Reply:
Same here.